cmd/exit_fetch.go

```diff
@@ -128,7 +128,7 @@ func runFetchExit(ctx context.Context, config exitConfig) error {
 
 			fullExit, err := oAPI.GetFullExit(valCtx, validatorPubKeyHex, cl.GetInitialMutationHash(), shareIdx, identityKey)
 			if err != nil {
-				return errors.Wrap(err, "could not load full exit data from Obol API")
+				return errors.Wrap(err, "load full exit data from Obol API", z.Str("validator_public_key", validatorPubKeyHex))
```

Comment:
Isn't the validator hex already in the context at this point?

Reply:
It is in the `valCtx`, but this error is unaware of it

Reply:
Ah right!